test/test_linalg.py

```diff
@@ -527,6 +527,42 @@ def run_test_case(input, ord, dim, keepdim, should_error):
                 for ord in ord_matrix:
                     run_test_case(input, ord, dim, keepdim, ord in error_ords)
 
+    def test_norm_deprecated(self, device):
+        expected_message = (
+            r'torch.norm is deprecated and may be removed in a future PyTorch release. '
+            r'Use torch.linalg.norm instead.')
+        for func in [torch.norm, torch.functional.norm]:
+            with self.maybeWarnsRegex(Warning, expected_message):
```

Comment:
Since this always warns the test can use a pattern like this:

https://github.com/pytorch/pytorch/blob/ba6ddaf04cebe29b8035f749c0796b5e185b5249/test/test_torch.py#L2055

and validate that two warnings are thrown.